src/lift.js

```diff
@@ -18,11 +18,11 @@ var liftN = require('./liftN');
  *
  *      var madd3 = R.lift((a, b, c) => a + b + c);
  *
- *      madd3([1,2,3], [1,2,3], [1]); //=> [3, 4, 5, 4, 5, 6, 5, 6, 7]
+ *      madd3([100, 200], [30, 40, 50], [6, 7]); //=> [136, 137, 146, 147, 156, 157, 236, 237, 246, 247, 256, 257]
  *
  *      var madd5 = R.lift((a, b, c, d, e) => a + b + c + d + e);
  *
- *      madd5([1,2], [3], [4, 5], [6], [7, 8]); //=> [21, 22, 22, 23, 22, 23, 23, 24]
+ *      madd5([10000, 20000], [3000], [400, 500], [60], [7, 8]); //=> [13467, 13468, 13567, 13568, 23467, 23468, 23567, 23568]
```

Comment:
We have `sanctuary` in the repl, how about something like

`madd3(S.Just(1), S.Just(2), Just(3)); //=> Just(6)` as a second example rather than `madd5`